lib/gap_intelligence/models/header.rb

```diff
@@ -0,0 +1,13 @@
+module GapIntelligence
+  class Header < Record
+    attribute :name
+
+    def initialize(attrs = {})
```

Comment:
@cheezy2022 I'm totally fine with that way it is now with headers changes so that we have consistency with existing code and talk/discuss that with @GapIntelligence/dev-team later on